### ISSUE TITLE
Flatpack crash before auth

### ISSUE THREAD

**Issue by Chu-4hun** [Vencord/Vesktop#446]
**Describe the bug**

fresh Flatpack install crashes on startup. rpm verrsion works fine. 

**To Reproduce**

**Expected behavior**

Open the app and login. 

**Screenshots**

https://github.com/Vencord/Vesktop/assets/70685495/4eacbe18-88ed-4c58-ac7b-c22cdb31e04d

**Desktop (please complete the following information):**
 - OS/Distro: Fedora Linux
 - Desktop Environment (linux only): kde
 - Version: 39 x86_64 
 - Vesktop version 1.5.1

**Command line output**

```
flatpak run dev.vencord.Vesktop
Passing the following arguments to Electron: --ozone-platform-hint=auto
[3:0315/171215.360716:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:0315/171215.599111:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:0315/171215.599135:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
(node:3) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/dan/.var/app/dev.vencord.Vesktop/config/vesktop/settings/quickCss.css'
(Use `vesktop.bin --trace-warnings ...` to show where the warning was created)
(node:3) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
[50:0315/171215.781187:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781341:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781394:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781442:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781500:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781559:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781600:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781625:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=SCANOUT
[50:0315/171215.781655:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781690:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781728:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781762:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781889:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781940:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.781999:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[50:0315/171215.782030:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=GPU_READ
[50:0315/171215.782047:ERROR:shared_image_factory.cc(984)] CreateSharedImage: could not create backing.
[50:0315/171215.782070:ERROR:shared_image_factory.cc(767)] DestroySharedImage: Could not find shared image mailbox
[50:0315/171215.782112:ERROR:gpu_service_impl.cc(1154)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[3:0315/171215.802942:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=8704
[92:0315/171215.874477:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874574:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874632:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874680:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874731:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874780:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874832:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.874865:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=SCANOUT
[92:0315/171215.874899:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875083:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875141:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875198:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875250:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875297:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875348:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[92:0315/171215.875373:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=GPU_READ
[92:0315/171215.875387:ERROR:shared_image_factory.cc(984)] CreateSharedImage: could not create backing.
[92:0315/171215.875404:ERROR:shared_image_factory.cc(767)] DestroySharedImage: Could not find shared image mailbox
[92:0315/171215.875444:ERROR:gpu_service_impl.cc(1154)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[3:0315/171215.890965:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=8704
[106:0315/171215.958198:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958319:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958407:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958484:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958547:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958611:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958676:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958720:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=SCANOUT
[106:0315/171215.958856:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958920:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.958991:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.959055:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.959118:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.959190:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.959254:ERROR:gbm_wrapper.cc(76)] Failed to get fd for plane.: No such file or directory (2)
[106:0315/171215.959308:ERROR:gbm_pixmap_wayland.cc(82)] Cannot create bo with format= RGBA_8888 and usage=GPU_READ
[106:0315/171215.959338:ERROR:shared_image_factory.cc(984)] CreateSharedImage: could not create backing.
[106:0315/171215.959370:ERROR:shared_image_factory.cc(767)] DestroySharedImage: Could not find shared image mailbox
[106:0315/171215.959440:ERROR:gpu_service_impl.cc(1154)] Exiting GPU process because some drivers can't recover from errors. GPU process will restart shortly.
[3:0315/171215.974377:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=8704
[79:0315/171216.031344:ERROR:command_buffer_proxy_impl.cc(131)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
Failed to import settings: Error: ENOENT: no such file or directory, scandir '/home/dan/.var/app/dev.vencord.Vesktop/config/Vencord/settings'
    at readdirSync (node:fs:1513:3)
    at t.readdirSync (node:electron/js2c/node_init:2:11289)
    at WebContents.<anonymous> (VCDMain:40:1004)
    at WebContents.emit (node:events:514:28) {
  errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: '/home/dan/.var/app/dev.vencord.Vesktop/config/Vencord/settings'
}
[arRPC > ipc] listening at /run/user/1000/discord-ipc-0
[arRPC > websocket] listening on 6463
[arRPC > process] started
```

**Additional context**

2 days before i was trying to login but stuck in reCAPCHA loop, but it was happening even on official client. After that i deleted Vesktop flatpack. Now im reviziting my installation and this is  what happend

**Comment by Vendicated** [comment]
based on your logs this is a gpu issue

**Comment by Vendicated** [comment]
for solutions, see other reports of the same issue, there are some discussions and solutions: https://github.com/search?q=repo%3AVencord%2FVesktop+RGBA_8888&type=issues

tldr: disable hardware acceleration or run with --disable-gpu, or try using appimage/rpm instead of flatpak
